l10n_ch_lsv_dd/model/banking_export_ch_dd.py

```diff
@@ -0,0 +1,95 @@
+##############################################################################
+#
+#    Swiss localization Direct Debit module for OpenERP
+#    Copyright (C) 2014 Compassion (http://www.compassion.ch)
+#    @author: Cyril Sester <cyril.sester@outlook.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, _
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class banking_export_ch_dd(models.Model):

```

Comment:
It is preferable to have CamelCase for class names.
I won't block for that thought